app/components/learnergroup-list.js

```diff
@@ -1,17 +1,47 @@
 import Component from '@ember/component';
+import { computed } from '@ember/object';
 
 export default Component.extend({
   classNames: ['learnergroup-list'],
 
+  bubbleSort: true,
```

Comment:
`learnergroup-list` component are used two separate times. The sort value needs to either be bubbled up to controller level for query param or stay within the component.

Reply:
Oh. Bubble as in bubble up. Not as in a type of sort. Ok. Thanks.

Reply:
I skipped course count sort because it's a lot more complicated. @dartajax 

```
<td class="text-center hide-from-small-screen">
  {{get (await learnerGroup.courses) "length"}}
</td>
```